joplin/base/signals/janis_build_triggers.py

```diff
@@ -19,11 +17,12 @@
 JANIS_SLUG_URL = settings.JANIS_SLUG_URL
 
 
-def trigger_build(sender, action='saved', instance=None):
+def trigger_build(sender, pages_ids, action='saved', instance=None):
```

Comment:
We should probably set this up on a feature flag, so we retain the ability to _not_ publish incrementally, plus that will let us merge the code into staging sooner